drivers/ieee802154/ieee802154_nrf5.c

```diff
@@ -393,7 +396,22 @@ static int nrf5_tx(const struct device *dev,
 		nrf_802154_transmit_csma_ca_raw(nrf5_radio->tx_psdu);
 		break;
 	case IEEE802154_TX_MODE_TXTIME:
-	case IEEE802154_TX_MODE_TXTIME_CCA:
+	case IEEE802154_TX_MODE_TXTIME_CCA: {
+		bool cca = (mode == IEEE802154_TX_MODE_TXTIME_CCA);
+		uint32_t tx_at;
+
+		__ASSERT_NO_MSG(pkt);
+		__ASSERT(IS_ENABLED(CONFIG_NET_PKT_TIMESTAMP),
+			 "Timestamp is required");
+		tx_at = net_pkt_timestamp(pkt)->second * USEC_PER_SEC
```

Comment:
You can reach txtime with `net_pkt_txtime`

Reply:
Done.
I'm closing the one, we can comment on symbol `CONFIG_NET_PKT_TXTIME` under the comment
https://github.com/zephyrproject-rtos/zephyr/pull/29123#discussion_r503263801